tests/test_e2e/test_actions.py

```diff
@@ -94,11 +86,34 @@ def handle_evt(charm: CharmBase, evt: ActionEvent):
     mycharm._evt_handler = handle_evt
 
     ctx = Context(mycharm, meta={"name": "foo"}, actions={"foo": {}})
-    out = ctx.run_action(ctx.on.action("foo"), State())
+    with pytest.raises(ActionFailed) as out:
+        ctx.run(ctx.on.action("foo"), State())
+    assert out.value.message == "failed becozz"
+    task = ctx.action_output
```

Comment:
I think it's weird calling this thing "task" when it's actually an ActionOutput. Can we call this variable `output`? (Here and elsewhere.)

Reply:
Not relevant any more (but I would have renamed - I just missed it when I lazily used the IDE to rename the class).